theme/themes/default/globals/site.variables

```diff
@@ -76,8 +76,8 @@
 
 /* This adjusts the default form input across all elements */
 @inputBackground        : @white;
-@inputVerticalPadding   : @relative11px;
-@inputHorizontalPadding : @relative14px;
+@inputVerticalPadding   : @relative1px;
```

Comment:
@lkttyagi are you sure this does not affect any other css styles in the site? You are changing some base settings here, not just a single css statement.

Reply:
yes  it doesnt change any other css styles.